src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderTypeConfirm.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿@using Dfc.CourseDirectory.WebV2.Features.Providers.EditProviderType
+@model ConfirmViewModel
+@{ 
+    ViewContext.SetLayoutData(title: "Update provider type");
+
+    var affectedItemCountErrors = ViewData.ModelState[nameof(ConfirmViewModel.AffectedItemCounts)]?.Errors
```

Comment:
Can't help but think there's a better way to do this?

Reply:
`var affectedItemCountError = Html.ValidationMessageFor(m => m.AffectedItemCounts);`

Reply:
I can't see how that would work? I don't believe this tag helper yields a collection of errors.